package.json

```diff
@@ -36,6 +36,7 @@
     "test": "pnpm run /^test:/"
   },
   "dependencies": {
+    "bun": "^1.1.17",
```

Comment:
No-no, it will install `bun` to _every_ Browserslist users.

At least put it to `devDependencies`.